### PR TITLE
Youtube Shorts in video player

### PR DESCRIPTION
**Pull request by kyellareddy**
## Description

This script opens the current Youtube Short in the normal Youtube video player.

It supports Safari and Chrome.

## Type of change

- [x] New script command

## Screenshot

https://github.com/raycast/script-commands/assets/68571978/2e79bb2d-0ad7-4957-9019-f94620baf080

## Dependencies / Requirements

None

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/browsing/Youtube Shorts in video player.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# @raycast.description Opens current Youtube Shorts video in the normal video player by replacing "/shorts/" with "/v/" in the url.
+# @raycast.author Kailash Yellareddy
+# @raycast.authorURL https://github.com/kyellareddy
+
+# -----------------------------------------------------------------
+# |  # This script currently supports Safari and Google Chrome.    |
+# |  # Replace "ABCD" in line 21 with either "Safari" or "Chrome"  |
+# -----------------------------------------------------------------
+
+set browser to "ABCD"
```
Comment (dehesa) [inline commands/browsing/Youtube Shorts in video player.sh:21]: I would recommend making the default `"Safari"` but left the comment above. This way the command would work by default.

**Review by dehesa — CHANGES_REQUESTED** [review]
Hi @kyellareddy,

Very useful script. Thanks! Just one note below.

File: commands/browsing/Youtube Shorts in video player.sh (hunk 1 above)
Comment (kyellareddy) [reply commands/browsing/Youtube Shorts in video player.sh:21]: Is there a way for me to make an edit to the code inside this pull request or do I have to make a new pull request with the change?

I don't have much experience with Github.

Comment (dehesa) [reply commands/browsing/Youtube Shorts in video player.sh:21]: There are a couple of ways. I believe you could even do it from the website, but what I usually do is just changed to code, create a new commit and push it.

Comment (kyellareddy) [reply commands/browsing/Youtube Shorts in video player.sh]: Oh, ok. I just wasn't sure if any new commits I made after I first made would update the pull request.

I made a new commit changing the default to `Safari`.

**Review by dehesa — APPROVED** [review]

**Pull request merged**